dafni_cli/get.py

```diff
@@ -46,13 +52,24 @@ def models(ctx: Context, creation_date: str, publication_date: str):
         if publication_date:
             date_filter = model.filter_by_date("publication", publication_date)
         if date_filter:
-            model.output_model_details()
+            model.output_model_details(long)
 
 
 @get.command()
+@click.argument("version-id", nargs=-1)
 @click.pass_context
-def metadata(ctx):
-    pass
+def model(ctx, version_id):
```

Comment:
is this actually the version id? I got really confused looking through through the models response as there isnt a version ID returned. I tried the ID field, and that seems to work, so should this instead be model ID?

Reply:
So I think it's unclear whether a new version of a model is a "new model" or a "new version". Because new versions of the same model will have a different ID field (I believe). And the ID of a model I think is just the version ID of the latest version (but I'll need to check this).

Reply:
But for user-friendliness, we could call it model_ID